runtime/fs_fake.js

```diff
@@ -213,7 +213,7 @@ MlFakeDevice.prototype.is_dir = function (name) {
   return this.content[name_slash] ? 1 : 0;
 };
 MlFakeDevice.prototype.unlink = function (name) {
-  var ok = this.content[name] ? true : false;
+  var ok = !!this.content[name];
```

Comment:
I find this syntax too cryptic, maybe we should use `hasOwnProperty` ..

Reply:
Indeed

Reply:
It's not really recommended and it's better to use `Object.hasOwn`, but that's a functionality added in ES2022, so how about revisiting it for subsequent PR to modernise?
https://biomejs.dev/linter/rules/no-prototype-builtins

Reply:
I'm still unconvinced by the change. Do we actually want to return a boolean..

Reply:
At this point, just reverted.